test/test_epoll.cpp

```diff
@@ -99,6 +121,39 @@ TEST(CEPoll, WaitEmptyCall)
     EXPECT_EQ(srt_cleanup(), 0);
 }
 
+TEST(CEPoll, WaitEmptyCall2)
```

Comment:
"2" as a suffix does not make much sense. Better to add "U" to provide a hint about `srt_uwait usage`.
Example: "UWaitEmptyCall"